_clients/data-prepper/data-prepper-reference.md

```diff
@@ -151,12 +152,35 @@ acmCertificateArn | Conditionally | String | Represents the ACM certificate ARN.
 
 ### string_converter
 
-Converts strings to uppercase or lowercase. Mostly useful as an example if you want to develop your own prepper.
+Converts string to uppercase or lowercase. Mostly useful as an example if you want to develop your own processor.
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
 upper_case | No | Boolean | Whether to convert to uppercase (`true`) or lowercase (`false`).
 
+### aggregate
+
+Groups events together based on the keys provided and performs a action on each group.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+identification_keys | Yes | List | A unordered list by which to group Events. Events with the same values for these keys are put into the same group. If an Event does not contain one of the `identification_keys`, then the value of that key is considered to be equal to `null`. At least one identification_key is required.
```

Comment:
Can you provide an example of what a list of `identification_keys` might look like? Even if it's just a list of string values, I think that could be helpful as a sort of template the customer can refer to.

Reply:
Sure, I can add an example here.